### PR TITLE
Simplify attach handling

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Attach can rely on broadcast channels as well as having dedicated tasks for reading and writing to the UNIX sockets. This allows us to drop the tick interval in container_io.

#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
@haircommander @rphillips PTAL

File: conmon-rs/server/src/attach.rs [hunk 1]
```diff
@@ -103,153 +143,147 @@ impl Attach {
...
+                    let (read, write) = stream.into_split();
+
+                    let tx_clone = tx.clone();
+                    task::spawn(
+                        async move {
+                            if let Err(e) = Self::read_loop(read, tx_clone).await {
+                                error!("Attach read loop failure: {:#}", e);
+                            }
+                        }
+                        .instrument(debug_span!("read_loop")),
```
Comment (haircommander) [inline conmon-rs/server/src/attach.rs:186]: just checking: will this say which read_loop we're in?

Comment (haircommander) [reply conmon-rs/server/src/attach.rs:186]: thought so, thanks for refreshing my memory :)

**Comment by haircommander** [comment]
/lgtm

good cleanup!

**Comment by rphillips** [comment]
Nice!

**Pull request merged**